config/locales/cy.yml

```diff
@@ -481,77 +544,377 @@ cy:
     followers_count: Nifer y dilynwyr
-    lock_link: Cloi eich cyfri
+    lock_link: Cloi eich cyfrif
+    purge: Dileu o dilynwyr
+    success:
+      one: Yn y broses o ysgafn-flocio dilynwyr o un parth...
+      other: Yn y broses o ysgafn-flocio dilynwyr o %{count} parth...
+    true_privacy_html: Cofiwch <strong>mai ond amgryptio pen-i-ben all sicrhau gwir breifatrwydd</strong>.
+    unlocked_warning_html: Gall unrhywun eich dilyn yn syth i weld eich tŵtiau preifat. %{lock_link} i gael adolygu a gwrthod dilynwyr.
     unlocked_warning_title: Nid yw eich cyfrif wedi ei gloi
   footer:
     developers: Datblygwyr
     more: Mwy…
     resources: Adnoddau
   generic:
     changes_saved_msg: Llwyddwyd i gadw y newidiadau!
+    copy: Copïo
     save_changes: Cadw newidiadau
     validation_errors: Mae rhywbeth o'i le o hyd! Edrychwch ar y %{count} gwall isod os gwelwch yn dda
   imports:
     preface: Mae modd mewnforio data yr ydych wedi allforio o achos arall, megis rhestr o bobl yr ydych yn ei ddilyn neu yn blocio.
-    success: Uwchlwyddwyd eich data yn llwyddiannus ac fe fydd yn cael ei brosesu mewn da bryd
+    success: Uwchlwythwyd eich data yn llwyddiannus ac fe fydd yn cael ei brosesu mewn da bryd
     types:
       blocking: Rhestr blocio
       following: Rhestr dilyn
       muting: Rhestr tawelu
     upload: Uwchlwytho
-  in_memoriam_html: ''
+  in_memoriam_html: In Memoriam.
   invites:
     delete: Dadactifadu
+    expired: Wedi darfod
     expires_in:
+      '1800': 30 munud
+      '21600': 6 awr
+      '3600': 1 awr
+      '43200': 12 awr
+      '604800': 1 wythnos
       '86400': 1 dydd
+    expires_in_prompt: Byth
+    generate: Cynhyrchu
+    invited_by: 'Cawsoch eich gwahodd gan:'
+    max_uses:
+      one: 1 defnydd
+      other: "%{count} defnydd"
     max_uses_prompt: Dim terfyn
     prompt: Cynhyrchwch a rhannwch ddolenni gyda eraill i ganiatau mynediad i'r achos hwn
     table:
+      expires_at: Darfod ar
       uses: Defnyddiau
+    title: Gwahodd pobl
+  lists:
+    errors:
+      limit: Yr ydych wedi cyrraedd uchafswm nifer y rhestrau posib
+  media_attachments:
+    validations:
+      images_and_video: Ni ellir ychwanegu fideo at statws sy'n cynnwys delweddau'n barod
+      too_many: Ni ellir ychwanegu mwy na 4 dogfen
   migrations:
-    currently_redirecting: ''
+    acct: enwdefnyddiwr@parth y cyfrif newydd
+    currently_redirecting: 'Mae eich proffil wedi ei osod i ailgyfeirio i:'
     proceed: Cadw
+    updated_msg: Diweddarwyd gosodiad mudo eich cyfrif yn llwyddiannus!
   moderation:
     title: Cymedroli
   notification_mailer:
+    digest:
+      action: Gweld holl hysbysiadau
+      body: Dyma grynodeb byr o'r holl negeseuon golloch chi ers eich ymweliad diwethaf ar %{since}
+      mention: 'Soniodd %{name} amdanoch chi:'
+      new_followers_summary:
```

Comment:
Maybe I think that need to organize 627-628, 630-631 lines as well.

It has not been pointed out in # 9104, but it has been pointed out in cs.yml .
https://github.com/tootsuite/mastodon/pull/9104/files#diff-2a5406712ed4bdf998f81b4ae9fb3ae1R615

Does anyone know about it?